ci_pod/jupyterlab/Dockerfile

```diff
@@ -4,18 +4,36 @@ ARG JUPYTERLAB_VERSION
 ENV JUPYTERLAB_VERSION=$JUPYTERLAB_VERSION
 ARG MAINTAINER
 ENV MAINTAINER=$MAINTAINER
-
 LABEL maintainer="Your Name <$MAINTAINER>"
+ARG OC_VERSION
+ENV OC_VERSION=$OC_VERSION
+ARG VIRTCTL_VERSION
+ENV VIRTCTL_VERSION=$VIRTCTL_VERSION
 
 # Install system dependencies
 RUN dnf update -y && \
     dnf install -y wget podman curl python3 python3-pip python3-devel nodejs npm && \
     dnf clean all && \
     rm -rf /var/cache/dnf && \
-    pip3 install --no-cache-dir jupyterlab=="$JUPYTERLAB_VERSION" benchmark-runner && \
+    pip3 install --no-cache-dir jupyterlab=="$JUPYTERLAB_VERSION" benchmark-runner bokeh && \
     jupyter labextension install @jupyter-widgets/jupyterlab-manager
 
+# install oc/kubectl client tools for OpenShift/Kubernetes
+RUN  curl -L https://github.com/openshift/okd/releases/download/$OC_VERSION/openshift-client-linux-$OC_VERSION.tar.gz -o  ~/openshift-client-linux-$OC_VERSION.tar.gz \
+     && tar -xzvf  ~/openshift-client-linux-$OC_VERSION.tar.gz -C  ~/ \
```

Comment:
Use $HOME rather than ~.  Also, quote everything that includes a variable that can be expanded.  E. g.

```
curl -L "https://github.com/openshift/okd/releases/download/${OC_VERSION}/openshift-client-linux-${OC_VERSION}.tar.gz" -o  "$HOME/openshift-client-linux-$OC_VERSION.tar.gz"
```